server/src/main/java/org/elasticsearch/search/aggregations/timeseries/TimeSeriesIndexSearcher.java

```diff
@@ -106,7 +106,7 @@ boolean next() throws IOException {
                 docId = iterator.nextDoc();
                 if (docId != DocIdSetIterator.NO_MORE_DOCS && (liveDocs == null || liveDocs.get(docId))) {
                     if (tsids.advanceExact(docId)) {
-                        BytesRef tsid = tsids.lookupOrd(tsids.nextOrd());
+                        BytesRef tsid = tsids.lookupOrd(tsids.ordValue());
```

Comment:
We can replace this with an ord comparison now that we know there's a single value, but let's do that in a follow-up.